source/extensions/filters/common/expr/library/custom_vocabulary.h

```diff
@@ -0,0 +1,45 @@
+#include "envoy/http/header_map.h"
+#include "envoy/stream_info/stream_info.h"
+
+#include "source/common/protobuf/protobuf.h"
+#include "source/extensions/filters/common/expr/context.h"
+
+#include "eval/public/cel_value.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Filters {
+namespace Common {
+namespace Expr {
+namespace Library {
+
+class CustomVocabularyWrapper : public BaseWrapper {
+public:
+  CustomVocabularyWrapper(Protobuf::Arena& arena, const StreamInfo::StreamInfo& info,
+                          const Http::RequestHeaderMap* request_headers,
+                          const Http::ResponseHeaderMap* response_headers,
+                          const Http::ResponseTrailerMap* response_trailers)
+      : arena_(arena), info_(info) {
+    request_headers_ = request_headers;
+    response_headers_ = response_headers;
+    response_trailers_ = response_trailers;
```

Comment:
Can we consistently use the initializer list here?

Reply:
ok